### PR TITLE
Update release process to include automations

### PR DESCRIPTION
**Pull request by jabrah**
Preview of the new page: https://github.com/eclipse-pass/main/blob/faab683f52e7ea43fbcbe1277b67eb202fb94f6e/docs/dev/release-steps-with-automations.md

ps: I have no idea why Markdown is putting a bullet point in front of one of the images 🙂

### CODE REVIEW

File: docs/dev/release.md [hunk 1]
```diff
@@ -89,8 +89,11 @@ The Java projects must follow a strict sequence, following its dependency hierar
    * [`pass-acceptance-testing`](https://github.com/eclipse-pass/pass-acceptance-testing)
 4. [`pass-docker`](https://github.com/eclipse-pass/pass-docker)
 
+These projects have GitHub workflow automations in place to perform releases that need to be triggered manually. See more detailed [release steps with automations](./release-steps-with-automations.md)
```
Comment (aforward) [inline docs/dev/release.md:92]: I prefer absolute paths so `[release steps with autmations](/docs/dev/release-steps-with-automations.md)` as it makes it easier to move files already and globally "search and replace" references to moved files.

File: docs/dev/release-steps-with-automations.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+      * [proxy](https://github.com/orgs/eclipse-pass/packages/container/package/proxy)
+      * [pass-ui-public](https://github.com/orgs/eclipse-pass/packages/container/package/pass-ui-public)
+      * [idp](https://github.com/orgs/eclipse-pass/packages/container/package/idp)
+      * [demo-ldap](https://github.com/orgs/eclipse-pass/packages/container/package/demo-ldap)
+
+## GitHub code release
+
+You will have to manually create a GitHub release through the GitHub web interface to complete the release of each of these components. This can generally be done any time after the release automation completes successfully, since the release will be made against a tag created by the automation.
+
+![Code release section](../assets/github/releases/code-release-section.png)
```
Comment (aforward) [inline docs/dev/release-steps-with-automations.md:50]: Same comment here

`Code release section`

File: docs/dev/release-steps-with-automations.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+* Navigate to the Actions tab in your target repository. Select the workflow you want to trigger. E.g. the Release workflow in `main`: https://github.com/eclipse-pass/main/actions/workflows/release.yml
+* Click on the `Run workflow` button
+* Input the branch you wish to run the release against and the desired "Release" and "Next development" versions
+  * Release version: full release version, e.g. 0.5.0. These versions should be regarded as immutable. These releases for Java projects cannot be updated or deleted.
+  * Next development version: snapshot or development versions, e.g. 0.6.0-SNAPSHOT (please use all capital letters for the SNAPSHOT suffix). These development versions are intended to be overwritten.
+* After a few seconds, a new workflow run should appear in the table with a yellow (in-progress) status dot. Clicking on that will allow you to monitor the run's progress by watching logs.
+
+It is recommended that you monitor the automation after triggering it to make sure it completes successfully before moving on to release the next component.
+
+![Release page screenshot](../assets/github/releases/main-release-page.png)
```
Comment (aforward) [inline docs/dev/release-steps-with-automations.md:71]: Same comment here

`Code release section`

**Review by aforward — APPROVED** [review]
Minor comment about using absolute paths for references.

**Comment by tsande16** [comment]
Looks good to me. Read through the documentation a couple times and it was easily understood. The example was helpful in understanding.

**Review by tsande16 — APPROVED** [review]
Looks good!

**Review by cshannon2 — APPROVED** [review]
Looks good to me

**Pull request merged**